doc/tasks/shell.md

```diff
@@ -30,7 +30,7 @@ parameters:
         shell:
             scripts:
                - script.sh
-               - ["./bin/command", "arg1", "arg2"]
+               - ["-c", "./bin/command", "arg1", "arg2"]
```

Comment:
i would still leave non `-c` call

question: `["-c", "foo", "arg1"]` or `["-c", "foo arg1"]` ?

Reply:
Valid argument. The `-c`option takes the next argument as input string.
You'll need to configure it as 2 parameters
Example: `sh -c "ls -al"`